tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1003 @@
+or destination range</td>
+<td><p><pre>
+10.0.0.0/8
+</pre></p>
+
+</td>
+<td>IP address range for the on-premises subnet.</td>
+</tr>
+<tr>
+<td>Name (Cloud Router)</td>
+<td><p><pre>
+vpn-scale-test-vendor-rtr
+</pre></p>
+
+</td>
+<td>For dynamic routing: Cloud Router name</td>
+</tr>
+<tr>
+<td>Name (BGP session)</td>
+<td><p><pre>
+bgp-peer1
+</pre></p>
+
+</td>
+<td>For dynamic routing: BGP session name.</td>
+</tr>
+<tr>
+<td>Name (gateway 1)</td>
+<td><p><pre>
+vpn-scale-test-<vendor-name>-gw-1
+</pre></p>
+
+</td>
+<td>The name for the first GCP VPN gateway.</td>
+</tr>
+<tr>
+<td>Name (tunnel 1)</td>
+<td><p><pre>
+vpn-scale-test-tunnel1
+</pre></p>
+
+</td>
+<td>The name for the first VPN tunnel for <code>vpn-scale-test-<vendor-name>-gw-1</code></td>
+</tr>
+<tr>
+<td>Forwarding rule</td>
+<td><p><pre>
+fr-esp
+</pre></p>
+
+</td>
+<td>For the ESP protocol</td>
+</tr>
+<tr>
+<td>Forwarding rule</td>
+<td><p><pre>
+fr-udp500
+</pre></p>
+
+</td>
+<td>For the UDP protocol, port 500</td>
+</tr>
+<tr>
+<td>Forwarding rule</td>
+<td><p><pre>
+fr-udp4500
+</pre></p>
+
+</td>
+<td>For the UDP protocol, port 4500</td>
+</tr>
+</tbody>
+</table>
+
+### IPsec parameters
+
+<vendor name><product name> supports the following IPsec parameters and ciphers for configuring the VPN gateways and tunnels as described in this document._  _For Cloud VPN, see the detailed list of [GCP-supported IKEv2 and IKEv1 ciphers](https://cloud.google.com/vpn/docs/concepts/advanced#supported_ike_ciphers).
+
+<This is a sample table. Use supported parameters and values>
+
+<table>
+<thead>
+<tr>
+<th><strong>Parameter</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IPsec Mode</td>
+<td>ESP+Auth Tunnel mode (Site-to-Site)</td>
+</tr>
+<tr>
+<td>Authentication protocol</td>
+<td>Pre-shared key</td>
+</tr>
+<tr>
+<td>Key exchange version</td>
+<td>IKEv2</td>
+</tr>
+<tr>
+<td>Start</td>
+<td>Auto</td>
+</tr>
+<tr>
+<td>Perfect Forward Secrecy (PFS)</td>
+<td>Group 16</td>
+</tr>
+<tr>
+<td>Dead Peer Detection (DPD)</td>
+<td>60 5 periodic</td>
+</tr>
+</tbody>
+</table>
+
+<This is a sample table. Use supported parameters and values for IKE phase 1 and phase 2>
+
+<table>
+<thead>
+<tr>
+<th><strong>Cipher Role</strong></th>
+<th><strong>Cipher</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Encryption (IPsec) Mode</td>
+<td>esp-aes 256 esp-sha-hmac</td>
+</tr>
+<tr>
+<td>Integrity</td>
+<td>sha256</td>
+</tr>
+<tr>
+<td>Diffie-Hellman (DH)</td>
+<td>Group 16</td>
+</tr>
+<tr>
+<td>Lifetime</td>
+<td>36,000 seconds (10 hours)</td>
+</tr>
+</tbody>
+</table>
+
+# GCP-side configuration
+
+This section covers configuring Cloud VPN using either dynamic routing with the BGP protocol (recommended) or static routing.
+
+There are two ways to create VPN gateways on GCP, using the Google Cloud Platform Console and the `[gcloud `command-line tool.](https://cloud.google.com/sdk/) This section provides instructions for both.
```

Comment:
need the tick marks around gcloud without space and inside the bracket.

Reply:
This URL is still not rendering properly. Need to change it like this:
\[`gcloud`command-line tool.]\(https://cloud.google.com/sdk/)

Reply:
Still needs to be done.